Packs/GitHub/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "GitHub",
     "description": "Manage GitHub issues and pull requests directly from Cortex XSOAR",
     "support": "xsoar",
-    "currentVersion": "1.1.11",
+    "currentVersion": "1.1.12",
```

this is an enhancement, bump the version by minor
```suggestion
    "currentVersion": "1.2.0",
```